tests/qchem/test_hamiltonians.py

```diff
@@ -108,135 +119,154 @@ def test_electron_integrals(symbols, geometry, core, active, e_core, one_ref, tw
 
 
 @pytest.mark.parametrize(
-    ("symbols", "geometry", "alpha", "h_ref"),
+    "use_jax",
     [
-        (
```

Comment:
Any reason the ref date is moved inside the function?

Reply:
Cleaner inside the function because we are not actually parametrizing across "symbols, geometries, alpha, h_ref". The alternative would be to copy and paste the reference data twice in the parametrize part just so that it can be parametrized across `use_jax`, which is much more redundant.